app/src/constants.ts

```diff
@@ -1,3 +1,4 @@
 export const surveyMonkeyUrl = 'https://www.surveymonkey.com/r/7BMHJL8'
 export const surveyMonkeyExitUrl = 'https://www.surveymonkey.com/survey-thanks'
+export const whereToUseWalletUrl = 'https://www2.gov.bc.ca/gov/content/governments/government-id/bc-wallet#where'
```

Comment:
Does this really get picked up by the ScanHelp screen or are you adding this for use later?

Reply:
Tested. Nope, not exposed. I've created a ticket to fix the problem because I'm not sure how to go about it and what to bring think through it a bit before jumping into a solution #1615.